autoload/sy.vim

```diff
@@ -106,7 +106,16 @@ endfunction
 
 " #buffer_is_active {{{1
 function! sy#buffer_is_active()
-  return !empty(getbufvar(bufnr(''), 'sy'))
+  return sy#buffer_is_active_bnr('')
```

Comment:
```suggestion
  return sy#buffer_is_active_bnr(bufnr(''))
```

Reply:
that's for looking like compatible, or new version viml actually donnot req ' '.